test/spec/common/index/test-cases/deleteBy.js

```diff
@@ -0,0 +1,96 @@
+'use strict';
+
+console.warn('lol it runs')
+
+module.exports = [
+  {
+    object: 'index',
+    methodName: 'deleteBy',
+    testName: 'index.deleteBy(undefined, cb)',
+    callArguments: [],
+    action: 'write',
+    expectedRequest: {
+      method: 'POST',
+      URL: {
+        pathname: '/1/indexes/%s/deleteByQuery',
+      },
+      body: {
+        params: '',
+      },
+    },
+  },
+  {
+    object: 'index',
+    methodName: 'deleteBy',
+    testName: 'index.deleteBy({query}, cb)',
+    callArguments: [{ query: 'some q' }],
+    action: 'write',
+    expectedRequest: {
+      method: 'POST',
+      URL: {
+        pathname: '/1/indexes/%s/deleteByQuery',
+      },
+      body: {
+        params: 'query=some%22q',
+      },
+    },
+  },
+  {
+    object: 'index',
+    methodName: 'deleteBy',
+    testName: 'index.deleteBy({query, filters}, cb)',
+    // this test should fail but it doesn't, so I don't trust the new tests 😭
```

Comment:
it doesn't enforce the error so I'm not sure what happens

Reply:
Why should it fails?

Reply:
I added `ridiculous:'mistake'` to the arguments to test if it was working, but it's not in the params and the test doesn't fail